### ISSUE TITLE
Team Speak APD Orginazation

### ISSUE THREAD

**Issue by OfficerShrek** [EasysAltisLife/Easys-Altis-Life#55]
I find that when a lot of cops log in, such as maybe 10, and all the cops are in Kavala Chat, things start to get hectic and commands are hard to follow. I want to suggest on this note Squad Chats, within District chats. Meaning inside the Kavala Chat category, there could be squad chats limited to three people. When Officers log into teamspeak they can quickly fill a squad chat, giving them a partner to role with, and a smoother, more organized police force. The highest ranking Officer in each chat/Squad Leader can communicate with the other higher ranking officers/leaders through the Arma group chat. With this, clear callouts could be made. And a proper command could take place. Leaders of group chats organize what their squads will do, then within the squads, easy co-operation can be made on the actions the leaders have agreed upon.

**Comment by imthatguyhere** [comment]
Unfortunately the idea is good but it falls apart in practice. There should never be 10 cops in kavala, often cops are there just to talk or are in another city / patrol and are hanging out.

As for squads, in game is the best way to handle that. You can team up with a few guys and roll. Use voice chat and ingame so you aren't clogging the channels and civs enjoy hearing banter between cops. Group chat ingame is too prone to breaking down to be used reliably.
